Justfile

```diff
@@ -0,0 +1,16 @@
+flox_bats_tests_invocation := "flox run '.#flox-tests' -- -- --flox target/debug/flox"
```

Comment:
I presume you are concerned about the FLOX_SH and FLOX_SH vars?

Reply:
Within a rust-env devshell, it won't!
If you want to do tests with a changed the bash code you need to build 

```
flox build flox-bash
export FLOX_SH_PATH="$PWD/result"
export FLOX_SH="$PWD/result/libexec/flox/flox"
```

Reply:
I can add the `flox build stuff` before the tests run.

The intent with this command is to (from inside the env) just run the whole test suite before pushing something so I don't have to wait 30 minutes to find out something is broken in CI. Since I rarely if ever touch the bash code (I think the first time I did this was yesterday) I don't see this as a huge problem.